### ISSUE TITLE
Mac build broken

### ISSUE THREAD

**Issue by cochrane** [opentomb/OpenTomb#114]
The Mac OS X build is largely broken right now. It is possible to generate an Xcode project from CMake if you have all the libraries installed, but that is not set up properly in many ways. I'm not yet sure if it is even able to produce a .app file instead of some Linux-specific thing. So we'll see how that one goes.

**Comment by vvs-** [comment]
I'd say that it depends on #115 and #119 then.

**Comment by cochrane** [comment]
Got an initial version working with some fudging (took me until Frankfurt/Main). In particular, I added the old OpenAL back in, because OS X standard OpenAL lacks all that effects stuff. Because of this, I put it into a branch for now. Internet in this train is sadly too spotty to do a push.

**Comment by stohrendorf** [comment]
Is it still broken?

**Comment by cochrane** [comment]
My branch isn't, but I haven't checked master lately

**Comment by stohrendorf** [comment]
Can this be closed now, or is the build still broken?

**Comment by cochrane** [comment]
I don't know, have you made a build and checked, or did any Mac user indicate that it can be closed (I haven't)? If yes, then you already know the answer. If not, then it needs to stay open.

**Comment by cochrane** [comment]
I checked, and no, the build still doesn't work. Which shouldn't be a big surprise given that this depends on #115, and that isn't resolved yet.

**Comment by stohrendorf** [comment]
I was just curious, as the last comment was from three weeks ago, and since then many things changed.

**Comment by cochrane** [comment]
No longer needed, references old version of the code.
